bot/commands/ticket.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from mongoengine.errors import DoesNotExist
+from telegram import Update
+from telegram.ext import CommandHandler, CallbackContext
+
+from bot.core import BotTelegramCore
+from bot.messages import GROUP_RESTRICTED, TX_ID_ERROR
+from db.observer import UserObserver
+from db.ticket import Ticket
+
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+
+def add_ticket(update: Update, context: CallbackContext):
+    chat = update.effective_chat
+    message = update.effective_message
+
+    if chat.type != "private":
+        message.reply_text(GROUP_RESTRICTED)
+        return
+
+    try:
+        observer = UserObserver.objects.get(chat_id=f"{chat.id}")
+    except DoesNotExist:
+        observer = UserObserver(f"{chat.username}",
+                                f"{chat.id}").save()
+
+    try:
+        tx_id = context.args[0]
+    except IndexError:
+        message.reply_text(TX_ID_ERROR)
+        return
+
+    try:
+        Ticket.objects.get(observer=observer, tx_id=tx_id)
+        message.reply_text(f"Ticket with transaction id {tx_id}"
+                           f" is already registered!", parse_mode='HTML')
+        return
+    except DoesNotExist:
+        ticket = Ticket(observer, tx_id)
+
+    ticket.fetch()
```

Comment:
If the ticket is not valid on this fetch it will be removed and then sent message saying "Your ticket has been saved".

Return `bool` on fetch to know if is valid or not before returning message.